### PR TITLE
Fix infinite loop when deep linking to hotspot.

### PR DESCRIPTION
**Pull request by matthewcarlreetz**
`helium://hotspot/:address`

### CODE REVIEW

File: src/features/hotspots/root/HotspotsView.tsx [hunk 1]
```diff
@@ -282,7 +282,8 @@ const HotspotsView = ({
     }
 
     fetchHotspot()
-  }, [dispatch, handlePresentDetails, params])
```
Comment (tyler-whitman) [inline src/features/hotspots/root/HotspotsView.tsx:285]: This used to work right? What changed?

Comment (matthewcarlreetz) [reply src/features/hotspots/root/HotspotsView.tsx:285]: I think handlePresentDetail now has a dependency that causes it to change when you deep link

Comment (matthewcarlreetz) [reply src/features/hotspots/root/HotspotsView.tsx:285]: which causes the useEffect to run again

Comment (tyler-whitman) [reply src/features/hotspots/root/HotspotsView.tsx:285]: Ah ok

**Review by tyler-whitman — APPROVED** [review]

**Pull request merged**